firebase-debug.1.log

```diff
@@ -0,0 +1,16 @@
+[debug] [2022-09-01T17:53:34.473Z] ----------------------------------------------------------------------
```

Comment:
You should not include the debug log in the PR because it can contain your computer information